src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -33,9 +33,20 @@ export interface SectionProps {
 export const Section: React.FC<SectionProps> = ({ venue }) => {
   const isReactionsAudioDisabled = !venue.isReactionsMuted;
 
-  const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide(
-    venue.isReactionsMuted ?? false
-  );
+  const {
+    isShown: isUserAudioOn,
+    toggle: toggleUserAudio,
+    hide: disableUserAudio,
+    show: enableUserAudio,
+  } = useShowHide(venue.isReactionsMuted ?? false);
```

Comment:
Shouldn't there already be a `DEFAULT_` constant somewhere? There is a `worldDefaultSettings.ts` to find these faster for the world. Maybe something similar should exist for the rest.

Reply:
added const + imported to use here e291c35